functional/SyntheticGCWorkload/playlist.xml

```diff
@@ -162,7 +162,7 @@
 		</variations>
 		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -cp $(Q)$(TEST_RESROOT)$(D)SyntheticGCWorkload.jar$(Q) net.adoptopenjdk.casa.workload_sessions.Main $(TEST_RESROOT)/config/config_doubleMapStress.xml -n; \
 	${TEST_STATUS}</command>
-		<!-- Temporarily disable this test on AArch64; github.com/eclipse/openj9/issues/8034 -->
+		<!-- Temporarily disable this test on AArch64; github.com/eclipse-openj9/openj9/issues/8034 -->
```

Comment:
While we are here, this now can be updated to use the new/preferred style for [temporarily disabling on a platform](https://github.com/adoptium/aqa-tests/blob/master/doc/userGuide.md#exclude-a-test-against-specific-platform).  Instead of the ^arch.aarch64 in `<platformRequirements>`, can add a `<disabled>` block at line 160:
```
<disabled>
	<comment>https://github.com/eclipse-openj9/openj9/issues/8034</comment>
	<plat>.*aarch64.*</plat>
</disabled>
```

Reply:
Sure, I'll do that.